kasa/experimental/sslaestransport.py

```diff
@@ -137,6 +137,11 @@ def default_port(self) -> int:
         """Default port for the transport."""
         return self.DEFAULT_PORT
 
+    @staticmethod
+    def _hash_credentials(credentials: Credentials) -> str:
```

Comment:
```suggestion
    def _create_b64_credentials(credentials: Credentials) -> str:
```
Or similar, perhaps? This differs from other hashes, so I'm wondering if we should document this also somehow?

Reply:
Done